toqito/matrix_props/kp_norm.py

```diff
@@ -16,16 +16,24 @@ def kp_norm(mat: np.ndarray, k: int, p: int) -> float:
     >>> import numpy as np
     >>> from toqito.matrix_props import kp_norm
     >>> from toqito.states import bell
-    >>> kp_norm(bell(0), 1, np.inf)
-    1.0
+    >>> '%.2f' % kp_norm(bell(0), 1, np.inf)
+    '1.00'
+
 
     To compute the k-largest singular values of a matrix:
 
     >>> import numpy as np
     >>> from toqito.matrix_props import kp_norm
     >>> from toqito.rand import random_unitary
-    >>> np.around(kp_norm(random_unitary(5), 5, 2), decimals=3)
-    2.236
+    >>> '%.2f' % kp_norm(random_unitary(5), 5, 2)
+    '2.24'
+
+    .. note::
+        You do not need to use `'%.2f' %` when you use this function.
+        We use this to format our output such that `doctest` compares the calculated output to the
+        expected output upto two decimal points only. The accuracy of the solvers can calculate the
+        `float` output to a certain amount of precision such that the value deviates after a few digits
+        of accuracy.
```

Comment:
I made this change here because the `doctest` workflow failed again. Pretty much the same thing  as what was done in 388eb87 but now we compare the strings not the value. 

https://github.com/vprusso/toqito/actions/runs/8582080338

Reply:
Appears to note have worked. Reopened #473